src/server/scripts/EasternKingdoms/ZulAman/boss_nalorakk.cpp

```diff
@@ -109,7 +110,7 @@ struct boss_nalorakk : public BossAI
 
     void MoveInLineOfSight(Unit* who) override
     {
-        if (who->IsPlayer() && _phase < PHASE_START_COMBAT && _active)
+        if (who->IsPlayer() && !who->ToPlayer()->IsGameMaster() && _phase < PHASE_START_COMBAT && _active)
```

Comment:
Is this needed? A gamemaster should not be able to get targeted in general

Reply:
Yes, it is required. Otherwise with `.gm on` the event will trigger.

I haven't tested with GM invis. But it does not make sense that GM's can be seen in the check below, with GM invis or without

https://github.com/azerothcore/azerothcore-wotlk/blob/9b5fc8775efce24f07cd81a45a248f60fc2171d5/src/server/game/Grids/Notifiers/GridNotifiers.cpp#L157